x/stakeibc/keeper/msg_server_liquid_stake.go

```diff
@@ -91,8 +83,8 @@ func (k msgServer) LiquidStake(goCtx context.Context, msg *types.MsgLiquidStake)
 func (k msgServer) MintStAsset(ctx sdk.Context, sender sdk.AccAddress, amount int64, denom string) error {
 	// repeat safety checks from LiquidStake in case MintStAsset is called from another site
 	// ensure Amount is positive, liquid staking 0 or less tokens is invalid
-	if amount < 1 {
-		k.Logger(ctx).Info("Amount to mint must be positive")
+	if amount <= 0 {
```

Comment:
Noting this is a logic fix Sam and I made, unrelated to the testing in the PR. Looks correct to me, pls take a look @vish-stride @asalzmann